client/app/components/project-milestone.js

```diff
@@ -11,11 +11,7 @@ export default class ProjectMilestoneComponent extends Component {
 
   @computed('milestone.statuscode')
   get getClassNames() {
-    let style = '';
-
-    if (this.milestone.statuscode === 'Not Started') style = 'gray';
-
-    if (this.milestone.statuscode === 'In Progress') style = 'green-glow';
+    const style = '';
 
     return `grid-x grid-padding-small milestone ${style}`;
   }
```

Comment:
Why set style at all?

Reply:
@dhochbaum-dcp not sure why it's like this. I probably has to do with the conditionals being removed. the whole computed should just be deleted and the classnames can be placed in-line.